L1Trigger/L1THGCal/src/backend/HGCalStage2ClusterDistribution.cc

```diff
@@ -0,0 +1,76 @@
+#include <unordered_set>
+#include <unordered_map>
+#include "L1Trigger/L1THGCal/interface/backend/HGCalStage2ClusterDistribution.h"
+#include "DataFormats/Common/interface/PtrVector.h"
+#include "DataFormats/Common/interface/OrphanHandle.h"
```

Comment:
It seems that many includes are not needed

Reply:
removed